src/posts/introducing-inference-data/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: 'Better Bayesian Workflows with Inference Data and Xarray'
+date: '2022-12-17'
+authors:
+  - name: Ravin Kumar
+    github: canyon289
+  - name: Oriol Abril-Pla
+    github: OriolAbril
+  - name: Osvaldo Martin
+    github: aloctavodia
+summary: "Xarray provides the core data model for ArviZ's InfrerenceData structure"
+---
+
+_TLDR: Xarray is the core of the `az.InferenceData` object. [ArviZ InferenceData](https://python.arviz.org/en/stable/api/generated/arviz.InferenceData.html) simplifies the Bayesian workflow, facilitates reproducibility, and enables interoperability between different Probabilistic Programming languages._
+
+<!-- Temporary Reference https://xarray.dev/blog/introducing-pint-xarray -->
```

Comment:
Remove HTML comments to fix build errors:

```console


12:28:08.086 | Running build in San Francisco, USA (West) – sfo1
-- | --
12:28:08.131 | Cloning github.com/xarray-contrib/xarray.dev (Branch: inf_data, Commit: 6bf3c39)
12:28:09.272 | Cloning completed: 1.140s
12:28:12.365 | Running "vercel build"
12:28:12.943 | Vercel CLI 28.16.6
12:28:13.331 | Installing dependencies...
12:28:13.710 | yarn install v1.22.17
12:28:13.828 | [1/4] Resolving packages...
12:28:14.226 | success Already up-to-date.
12:28:14.242 | Done in 0.54s.
12:28:14.261 | Detected Next.js version: 13.1.6
12:28:14.263 | Running "yarn run build"
12:28:14.493 | yarn run v1.22.17
12:28:14.553 | $ next build
12:28:15.253 | info  - Linting and checking validity of types...
12:28:17.992 | info  - Creating an optimized production build...
12:28:18.028 | warn  - /api/github provided runtime 'experimental-edge'. It can be updated to 'edge' instead.
12:28:32.537 | warn  - You are using an experimental edge runtime, the API might change.
12:28:45.596 | info  - Compiled successfully
12:28:45.597 | info  - Collecting page data...
12:28:50.069 | info  - Generating static pages (0/18)
12:28:50.773 | info  - Generating static pages (4/18)
12:28:50.873 | info  - Generating static pages (8/18)
12:28:51.373 |  
12:28:51.373 | Error occurred prerendering page "/blog/introducing-inference-data". Read more: https://nextjs.org/docs/messages/prerender-error
12:28:51.373 | Error: [next-mdx-remote] error compiling MDX:
Expand 13 Lines
12:28:52.025 | > Build error occurred
12:28:52.028 | Error: Export encountered errors on following paths:
12:28:52.028 | /blog/[id]: /blog/introducing-inference-data
12:28:52.028 | at /vercel/path0/node_modules/next/dist/export/index.js:415:19
12:28:52.029 | at runMicrotasks (<anonymous>)
12:28:52.029 | at processTicksAndRejections (internal/process/task_queues.js:95:5)
12:28:52.029 | at async Span.traceAsyncFn (/vercel/path0/node_modules/next/dist/trace/trace.js:79:20)
12:28:52.029 | at async /vercel/path0/node_modules/next/dist/build/index.js:1400:21
12:28:52.030 | at async Span.traceAsyncFn (/vercel/path0/node_modules/next/dist/trace/trace.js:79:20)
12:28:52.030 | at async /vercel/path0/node_modules/next/dist/build/index.js:1259:17
12:28:52.030 | at async Span.traceAsyncFn (/vercel/path0/node_modules/next/dist/trace/trace.js:79:20)
12:28:52.030 | at async Object.build [as default] (/vercel/path0/node_modules/next/dist/build/index.js:66:29)
12:28:52.106 | error Command failed with exit code 1.


```

```suggestion
```